extensions/pkg/controller/backupbucket/actuator.go

```diff
@@ -12,10 +12,18 @@ import (
 	extensionsv1alpha1 "github.com/gardener/gardener/pkg/apis/extensions/v1alpha1"
 )
 
-// Actuator acts upon BackupBucket resources.
+// Actuator acts upon [extensionsv1alpha1.BackupBucket] resources.
 type Actuator interface {
-	// Reconcile reconciles the BackupBucket.
+	// Reconcile reconciles the [extensionsv1alpha1.BackupBucket] resource.
+	//
+	// Implementations should take care for creating or updating the
+	// BackupBucket in order to bring it to its desired state.
```

Comment:
```suggestion
	// Implementations should ensure that the backup bucket is created or updated to reach its desired state.
```

It might be less confusing to not reuse the resource name here.

Reply:
Agree, a fix is on the way :)